### PR TITLE
Multiple in- and exclude patterns

### LINKED ISSUE #54: Command line parser doesn't support multiple occurrences of --include-pattern or --exclude-pattern
If **--include-pattern** or **--exclude-pattern** are given more than once only the last occurrence is taking effect, all others are effectively ignored.

This is sort of counter intuitive. I expected the following invocation:

`pss --include-pattern=A --include-pattern=B`

to be equivalent to

`pss --include-pattern='(A|B)'`

I know that since _pss_ supports Python's regular expression syntax this seems like a mere question of preference, but there exist use cases where the combination behavior is desired.

**Example**
I use vim (neovim) as my editor of choice. Vim supports the _grepprg_ option, that allows to specify an external program to be used for vim's grep command. I prefer to use _pss_ for this, because I am more familiar with PCRE. In addition it is possible to provide the command with command line options that will always be used. Since I develop Qt applications I am dealing with a lot of auto-generated files of the form **moc_.*cpp**. These files are only cluttering the search results so I want them excluded. The problem is when I provide this exclude pattern as part of the default command invocation it is not possible to use **--exclude-pattern** again.
So I am left with one of two options:

1. Write --exclude-pattern='moc_.*cpp' everytime I search something
2. Accept that my search results are cluttered when I use another exclude pattern in a search

### PR DESCRIPTION
**Pull request by TobiasPleyer**
Fixes #54 

The options **-g** **-G/--include-pattern** and **--exclude-pattern** can now be given more than once. If more than one pattern is given, the patterns are combined into one via regular expression "or" ('|').

### CODE REVIEW

**Comment by TobiasPleyer** [comment]
I discovered that the FileFinder class already supports multiple regular expressions. So the main task was to change the option parser so that it would parse a list of values of the relevant options instead of a single value.

**Comment by TobiasPleyer** [comment]
Hi and thanks for your feedback.
I wrote new unit tests for **test_driver** and **test_pssmain**.
In **test_driver** I experimented a bit with a more concise way to write the file matching tests. Let me know what you think about it. If you don't like it I will rewrite it in the way it was written before.
Regards

File: test/utils.py [hunk 1]
```diff
@@ -12,6 +12,13 @@
 from psslib.outputformatter import OutputFormatter
 
 
+def concat(*args):
```
Comment (eliben) [inline test/utils.py]: This function doesn't appear to do much, only used to save +s in call to sort? Not sure how I feel about this. Maybe wrap `sorted` as well - something like `sortedseq` that takes any number of iterables, concatenates them and calls sorted?

Comment (TobiasPleyer) [reply test/utils.py]: I agree. `concat` doesn't bring much of a win. I will completely get rid of this function.

**Pull request merged**